package.json

```diff
@@ -49,7 +49,6 @@
     "husky": "^4.2.5",
     "lint-staged": "^10.2.6",
     "prettier": "^2.0.5",
-    "typescript": "^3.8.3",
```

Comment:
coc-tsserver was using this installation of typescript instead of the one in `frontend`. I don't see this being used anywhere, so I removed it.